R/load_data.R

```diff
@@ -30,7 +30,12 @@
 #'   recent update before that date is used).
 #' @param na_rm (logical) Specifies whether to filter the data to only rows
 #'   where `value` is not missing. Defaults to `TRUE`.
-#' @param sandbox (logical) Whether or not to use the sandbox folders in the data lake.
+#' @param experiment (string) Either NULL (default) or a string.
```

Comment:
`date_filter` should be `version` in the documentation. I changed this in commit

Reply:
Addressed in commit f228ae7.